.github/workflows/rebase.yml

```diff
@@ -0,0 +1,20 @@
+name: Automatic Rebase
+on:
+  issue_comment:
+
+jobs:
+  rebase:
+    name: Rebase
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin rebase') && github.event.comment.author_association == 'MEMBER'
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
could you add a extra set to react to the rebase comments such as in the [jellyfin-vue repo](https://github.com/jellyfin/jellyfin-vue/blob/master/.github/workflows/rebase.yml#L11)?

Reply:
Done!

Also changed the command to ``@jellyfin-bot rebase`` as in Vue.